tests/test_easing.h

```diff
@@ -0,0 +1,166 @@
+/*************************************************************************/
+/*                       This file is part of:                           */
+/*                           GODOT ENGINE                                */
+/*                      https://godotengine.org                          */
+/*************************************************************************/
+/* Copyright (c) 2007-2020 Juan Linietsky, Ariel Manzur.                 */
+/* Copyright (c) 2014-2020 Godot Engine contributors (cf. AUTHORS.md).   */
+/*                                                                       */
+/* Permission is hereby granted, free of charge, to any person obtaining */
+/* a copy of this software and associated documentation files (the       */
+/* "Software"), to deal in the Software without restriction, including   */
+/* without limitation the rights to use, copy, modify, merge, publish,   */
+/* distribute, sublicense, and/or sell copies of the Software, and to    */
+/* permit persons to whom the Software is furnished to do so, subject to */
+/* the following conditions:                                             */
+/*                                                                       */
+/* The above copyright notice and this permission notice shall be        */
+/* included in all copies or substantial portions of the Software.       */
+/*                                                                       */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,       */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF    */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.*/
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY  */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,  */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE     */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
+/*************************************************************************/
+
+#ifndef TEST_EASING_H
+#define TEST_EASING_H
+
+#include "scene/animation/tween.h"
+
+namespace TestEasing {
+
+class MockTween : public Tween {
+public:
+	real_t run_equation(TransitionType p_trans_type, EaseType p_ease_type, real_t t, real_t b, real_t c, real_t d) {
+		return Tween::_run_equation(p_trans_type, p_ease_type, t, b, c, d);
+	}
+};
+
+const float TIME_START = 0;
+const float TIME_END = 1;
+const int STEPS = 30;
+
+/* clang-format off */
+const float expected[Tween::TRANS_COUNT][Tween::EASE_COUNT][STEPS+1] = {
+	{
+		{300.000000,316.666656,333.333344,350.000000,366.666687,383.333344,400.000000,416.666687,433.333344,450.000000,466.666687,483.333344,500.000000,516.666687,533.333374,550.000000,566.666687,583.333374,600.000000,616.666687,633.333374,650.000000,666.666687,683.333374,700.000000,716.666687,733.333374,750.000000,766.666687,783.333374,800.000000,},
```

Comment:
This is pretty minor, but I'd still separate each value in the array with a space after the comma.